src/com/rcv/CVRReader.java

```diff
@@ -220,4 +236,31 @@ private String getStringFromCell(Cell cvrDataCell) {
 
     return cellString;
   }
+
+  static class UnrecognizedCandidateException extends Exception {
```

Comment:
Do we need to add the function / purpose / param / returns comments for these two exception classes?

Reply:
Humble proposal: let's pause our practice of adding useless comments unless/until we get confirmation (from Paul Craft, I guess) that we actually need to do this. It's super tedious, it pollutes the code, and it will be easy to add later if necessary.

Reply:
Totally fine with me. Do you remember where you guys heard it was necessary? Maybe we should ask Paul directly?

Reply:
I forget who it was, but we spoke to someone else who was familiar with the code certification process a long time ago. Yeah, we should confirm all of this stuff with Paul.